services/masterdata/src/lib/components/masterdatas-edit-form/masterdatas-edit-form.component.html

```diff
@@ -1,5 +1,9 @@
 <lens-panel
-	header="{{ formTitle }} Masterdata to type '{{
+	header="{{
```

Comment:
suggestion: move to ts file (same for other occurrences in this file

Reply:
Good point! I've updated the code.